dbms/include/DB/Columns/ColumnVector.h

```diff
@@ -265,6 +265,16 @@ class ColumnVector final : public IColumn
 		res = typename NearestFieldType<T>::Type(data[n]);
 	}
 
+	const T & get(size_t n) const
```

Comment:
`get` name could be easily confused with another method with same name, that does different thing.
Try to think about different name...

Also note, that getData()[n] is not much more complex.